web3/content_script.js

```diff
@@ -105,6 +105,7 @@ function main () {
         origin
       }, (result) => {
         if (!result) {
+          retry(ev);
```

Comment:
> The JSON response object sent by the handler of the message. If an error occurs while connecting to the extension, the callback will be called with no arguments and runtime.lastError will be set to the error message.

I've seen no results here right after refreshing the page, not sure what's causing this. Will check `runtime.lastError` for some details, but perhaps the background script is not ready yet?